packages/typo3-docs-theme/assets/sass/components/_pagination.scss

```diff
@@ -0,0 +1,10 @@
+.page-link {
+    color: inherit;
+    background: #f2f2f2;
+    &:hover {
```

Comment:
We are using scss, so can be written as:

    &:hover,
    &:focus  {
         color: inherit;
    }

Then the property is only in one place.

Reply:
regarding accessibility focus and hover should not be the same, but focus should have a general styling like a box or some such?

Reply:
@linawolf you're right. It would be interesting to create a separated pull request "focused on focus". We could give the outlined style to all focused link in a more global way.

Reply:
In my adjustment I was now also tackling the focus. We have to have a look at the main menu focus-wise but should do that in a follow up